tools/DevTools/CommonRepo.psm1

```diff
@@ -0,0 +1,96 @@
+﻿<#
+ .Synopsis
+  Connects azure-powershell repo to azure-powershell-common repo for debugging.
+
+ .Description
+  Connects azure-powershell repo to azure-powershell-common repo for debugging.
+
+ .Parameter CommonRepoPath
+  Path to the common repo. Relative or absolute.
+
+ .Example
+   Connect-CommonRepo -CommonRepoPath ..\azure-powershell-common
+#>
+function Connect-CommonRepo {
+  [CmdletBinding()]
+  param(
+    [Parameter(Mandatory)]
+    [system.string]
+    ${CommonRepoPath}
+  )
+
+  Write-Host "1/2 Adding common projects to sln and csproj"
+
+  $csprojs = @(
+    '.\src\Authentication.Abstractions\Authentication.Abstractions.csproj',
+    '.\src\Common\Common.csproj',
+    '.\src\ResourceManager\ResourceManager.csproj',
+    '.\src\Graph.Rbac\Graph.Rbac.csproj'
+  )
+  $CommonRepoPath = (Resolve-Path $CommonRepoPath).Path
+  $RepoRoot = "$PSScriptRoot/../.."
+  Push-Location "$RepoRoot/src/Accounts"
+  try {
+    foreach ($csproj in $csprojs) {
+      $csproj = [System.IO.Path]::Combine($CommonRepoPath, $csproj)
+      $csproj = [System.IO.Path]::GetFullPath($csproj)
+      dotnet sln add $csproj
+      if ($LASTEXITCODE -ne 0) {
+        throw "Failed to add $csproj to Accounts.sln"
+      }
+      dotnet add ./Accounts/Accounts.csproj reference $csproj
```

Comment:
How do I update the script to support connecting common repo to not only Az.Accounts but also service modules like Az.KeyVault?

Reply:
Consider `Connect-CommonRepo -Solution ./src/KeyVault/KeyVault.sln -Csproj ./src/KeyVault/KeyVault.csproj`. By default Solution=Accounts.sln, -Csproj=Authentication.proj

Reply:
Does current script support connecting service modules?